filters/fasttext_filter.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+import os
+import sys
+import argparse
+from typing import List
+
+# The filename has to have a different name than 'fasttext', otherwise we can't import the module correctly
+import fasttext
+
+from more_itertools import chunked
+import requests
+
+
+def download_model(model_type: str):
+    """
+    Downloads the fasttext model for language identification to local directory. Either the large one or the small one.
+    """
+    if model_type == "small":
+        url = "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.ftz"
+    elif model_type == "large":
+        url = "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin"
+    else:
+        raise TypeError("Fasttext model type has to be either 'small' or 'large'.")
+    file_name = model_type + ".bin"
```

Comment:
What it be ok if the model is the 'ftz' model is renamed to `bin` here?

Reply:
Actually yes, they are a bit wild with the naming. 
But when I used the large model, I noticed that it's being downloaded very slowly, so I switched from streaming download to a normal one. It's 120MB so I think it should be fine.